### ISSUE TITLE
Server mode causing segmentation faults

### ISSUE THREAD

**Issue by ftuffner** [gridlab-d/gridlab-d#1440]
Initial Checklist prior to submitting the issue. Check the bellow boxes to the best of your knowledge:

- [X] I have not found an open issue describing the same problem.
- [X] I went through "CONTRIBUTING.MD" before deciding to submit the issue.
- [X] While building GridLAB-D, I followed all steps as detailed in GridLAB-D Wiki.

------------------------------------------------------------------------------------------------------------

## Please describe what was the cause of the issue?
Run a simple model in server mode (realtime or not) and it either drops a segmentation fault or silent fail.

## Please provide the steps to reproduce.
See the forum post at [this SourceForge post](https://sourceforge.net/p/gridlab-d/discussion/842562/thread/fd6363a478/)

Basically run `gridlabd --server IEEE_13_Node_With_Houses.glm` (located under the models directory).

## Please provide the exact error/warning/failure message printed on the console.
See the screenshots in the SourceForge post.

-------------------------------------------------------------------------------------------------------------
